### DISCUSSION TITLE
numba errors running python -m aquacrop.scripts.initiate_library

### DISCUSSION THREAD

**Discussion by AdinAdler** [aquacropos/aquacrop#161] [Q&A]
When I run the package in the development environment everything works as expected, but when I run `python -m aquacrop.scripts.initiate_library` I get the following error: 

```
lib/python3.10/site-packages/numba/core/datamodel/manager.py", line 36, in lookup
    handler = self._handlers[type(fetype)]
KeyError: "Failed in nopython mode pipeline (step: native lowering)\n<class 'type'>"
```
Half of the 20 package compilations fail this way. I've tried a full reinstall and made sure my packages match requirements.txt.

Did something change with the recent release that could cause this?

**Comment by tfoster88** [comment]
Hmm that is strange @AdinAdler. What version of the model are you using? Is this error still persisting? We haven't had much time to look into the issues with the compiler recently, but are aiming to do so in the coming weeks. As you seem to have already found, running in development environment is a fix in the meantime but obviously comes with a computational cost. I think we may need to look at alternatives to numba AOT compilation in time, but that's a larger job that may take a while to complete
